### DISCUSSION TITLE
Start Bot with Deck Setup Idea

### DISCUSSION THREAD

**Discussion by PeanutJar-Jelly** [AdriaGual/marvel-snap-bot#85] [Ideas]
It feels like as the library of cards grows that the bot has to scan, it gets slower and less successful to return positive ID's of the cards in hand.  My idea is to use the same framework but start the bot, passing in the deck and meta data that it should expect to see/scan since the decks don't change on a given run.  Giving it a deck of cards that it is expecting, I think would take less time to read through all cards and there would be no false positives.  The only setback using this method is that when cards are generated, the bot would probably not play those or know what to do with them.

Also, giving it more thorough instructions on how to play the deck per deck build rather than changing the master code since a cards purpose varies with the deck build.  Creating these for some of the main meta decks might actually yield a decent win percentage rather than just for farming purposes.

**Comment by paperc07** [comment]
so what can you do to test it out?
